lib/matplotlib/__init__.py

```diff
@@ -918,6 +1086,7 @@ def _rc_params_in_file(fname, transform=lambda x: x, fail_on_error=False):
 or from the matplotlib source distribution""",
                          dict(key=key, fname=fname, line_no=line_no,
                               line=line.rstrip('\n'), version=version))
+    config._new_child()
```

Comment:
As above: Please explain what this does and why we need it (at least in its docstring). The need to call a private method feels like a design flaw to me.